src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/GetTaskByIdControllerTest.java

```diff
@@ -21,14 +23,16 @@
 public class GetTaskByIdControllerTest extends SpringBootFunctionalBaseTest {
 
     private static final String ENDPOINT_BEING_TESTED = "task/{task-id}";
+    private static final String ENDPOINT_COMPLETE_TASK = "task/{task-id}/complete";
+    private String taskId;
 
     @Test
     public void should_return_a_404_if_task_does_not_exist() {
-        String nonExistentTaskId = "00000000-0000-0000-0000-000000000000";
+        taskId = "00000000-0000-0000-0000-000000000000";
```

Comment:
Personal opinion, I'd rather use the "nonExistentTaskId" since it describes more the intention and it's easier to understand what's going on. As I say, this is just a suggestion. I am happy with this change.

Reply:
I agree. I had to change the label for reuse.

Reply:
Not to be fussy, but I can see why you've done it like that in order to clean up the task in camunda however for this one we are not creating a task so I think I also agree with David on having a more descriptive intention :D

Reply:
The tearDown test uses taskId, in this case it will be null so completeTask endpoint fails. Alternatively I can do a null check in teardown for taskId but I feel it's unnecessary. IMHO, the test name describes what is tested. There is a tradeoff which way to go.